### ISSUE TITLE
Feature: Timer Visible in Notifications

### ISSUE THREAD

**Issue by AdarshRawat1** [CCExtractor/ultimate_alarm_clock#294]
## Problem - 
Currently the timer can only be checked via application 

## Solution - 
The timer should be visible in the app's push notifications, allowing users to quickly view the remaining time without opening the app. The notification should update in real-time as the timer counts down.

## Impact - 
This feature will enhance user convenience and improve the overall user experience. 

If you think this is a valid point, then allow me to work on it.

**Comment by MarkisDev** [comment]
Yup, I'll assign this to you. Just make sure it doesn't drain too much battery, we've had this issue in the past with persistent nofications (via a  foreground service)
